### PR TITLE
[CI]designate CUDA version when run lit in gh actions

### PR DESCRIPTION
**Pull request by DoyleLi**
Signed-off-by: Xiaodong Li <xiaodong.li@intel.com>

### CODE REVIEW

File: .github/workflows/ci_win.yml [hunk 1]
```diff
@@ -88,10 +88,30 @@ jobs:
         with:
           name: SYCLomatic_Win_${{ github.sha }}
           path: c2s_obj.7z 
+
       - name: Lit
         run: |
           cd c2s.src/build
           ninja check-clang-c2s -j8
```
Comment (delecui) [inline .github/workflows/ci_win.yml:98]: Windows "Lit" runs with CUDA 10.2. This is different from Linux "Lit", which runs with CUDA 11.6. 
Please investigate to fix it or set CUDA_PATH to 11.6 explicitly as temporary workaround.

File: .github/workflows/ci_win.yml [hunk 2]
```diff
@@ -88,10 +88,30 @@ jobs:
...
+        shell: bash
+
+      - name: Lit_CUDA8.0
+        run: |
+          cd c2s.src/build
+          set CUDA_PATH="L:/ref/cuda/win/v8.0" && ninja check-clang-c2s -j8
```
Comment (delecui) [inline .github/workflows/ci_win.yml]: `set CUDA_PATH="L:/ref/cuda/win/vM.N"` does not seems to work. CUDA 10.2 is still used.

Suggest you remove the quotes and have a try.

**Review by delecui — CHANGES_REQUESTED** [review]

**Comment by DoyleLi** [comment]
Hi @delecui and @zhimingwang36, the patch to enable Lit for different CUDA versions is ready. Please help to approve and merge the PR, thanks!

**Review by delecui — APPROVED** [review]
LGTM

**Comment by DoyleLi** [comment]
@zhimingwang36  could you please help to approve and merge the PR? Thank you!

**Review by zhimingwang36 — DISMISSED** [review]
LGTM

**Comment by DoyleLi** [comment]
@zhimingwang36 Ping.

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**